playbook.yml

```diff
@@ -193,26 +195,27 @@
       shell: oc create -f {{ tsb_subscription }}
 
     - name: Wait for templateservicebrokers resource type to exist
-      action: shell oc get crd | grep templateservicebrokers
-      register: cmd_result
-      until: cmd_result.rc == 0
-      retries: 10
-      failed_when: cmd_result.rc not in [0,1]
+      shell: oc get crd templateservicebrokers.osb.openshift.io
+      retries: 40
```

Comment:
Does this take 40 retries now? That seems like it might be a long time.

Reply:
No. I can reduce it. It was sometimes taking a long time with master, but since it does not work and it's much faster with 0.9.1 I can reduce it.